packages/commands/src/components/command-menu.js

```diff
@@ -126,18 +131,21 @@ export function CommandMenu() {
 	}, [] );
 	const [ loaders, setLoaders ] = useState( {} );
 
-	// Toggle the menu when Meta-K is pressed
 	useEffect( () => {
-		const toggleOnMetaK = ( e ) => {
-			if ( e.key === 'k' && e.metaKey ) {
-				setOpen( ( prevOpen ) => ! prevOpen );
-				e.preventDefault();
-			}
-		};
+		registerShortcut( {
+			name: 'core/commands',
+			category: 'global',
+			description: __( 'Open the global command menu' ),
+			keyCombination: {
+				modifier: 'primary',
+				character: 'k',
+			},
+		} );
+	}, [ registerShortcut ] );
 
-		document.addEventListener( 'keydown', toggleOnMetaK );
-		return () => document.removeEventListener( 'keydown', toggleOnMetaK );
-	}, [] );
+	useShortcut( 'core/commands', () => setOpen( ( prevOpen ) => ! prevOpen ), {
```

Comment:
@youknowriad Is there a way to prevent the default event? In Windows Firefox, ctrl+k jumps you to the search field in the browser, outside of the site editor.

Reply:
This should be fixed.